pykern/fconf.py

```diff
@@ -206,6 +206,8 @@ def __init__(self, files, base_vars=None):
             except Exception as e:
                 pykern.pkinspect.append_exception_reason(e, f"fconf.Parser.file={f}")
                 raise
+        if "yml" not in self.files:
+            raise ValueError("must supply at least on '.yml' file")
```

Comment:
Did you mean "at lease **one** '.yml' file"

Reply:
edbdcb5